code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -21,6 +21,16 @@ AI MODULES
 	origin_tech = list(TECH_DATA = 3)
 	var/datum/ai_laws/laws = null
 
+/obj/item/aiModule/attack_self(mob/user)
+	. = ..()
+	show_laws(user)
+
+/obj/item/aiModule/proc/show_laws(mob/user)
+	if(laws?.inherent_laws?.len)
+		to_chat(user, "<b>Programmed Law[(laws.inherent_laws.len > 1) ? "s" : ""]:</b>")
```

Comment:
Use macro

Reply:
It doesn't use any span classes. Just bold text.

Reply:
You are correct, which is why I want that to be changed to the span specifically designed for bold text.

https://github.com/vlggms/tegustation-bay12/blob/afb9fdcc58e40895df0ce0538d70c871c6f10ea6/code/_macros.dm#L170